config/_default/menus.en.toml

```diff
@@ -17,72 +17,72 @@ url = 'en/events/'
 weight = 30
 
 [[main]]
-name = "Minor Swing"
+name = "Minor Swing Social"
 url = "en/minorswing/"
 parent = "events"
 weight = 31
 
-[[main]]
-name = "Friday Night Balboa"
-url = "en/fridaynightbalboa/"
-parent = "events"
-weight = 32
-
-[[main]]
-name = "Mattstedt"
-url = "en/mattstedt/"
-parent = "events"
-weight = 33
-
-[[main]]
-name = "Tag der offenen Tür"
-url = "en/opendoor/"
-parent = "events"
-weight = 34
-
-[[main]]
-name = "Swing Tea Time"
-url = "en/swingteatime/"
-parent = "events"
-weight = 35
-
-[[main]]
-name = "Swing Brunch"
-url = "en/swingbrunch/"
-parent = "events"
-weight = 35
+#[[main]]
+#name = "Friday Night Balboa"
+#url = "en/fridaynightbalboa/"
+#parent = "events"
+#weight = 32
+#
+#[[main]]
+#name = "Mattstedt"
+#url = "en/mattstedt/"
+#parent = "events"
+#weight = 33
+#
+#[[main]]
+#name = "Open Doors Day"
```

Comment:
I guess it is just "Open Day" or "Open House".

Reply:
Yeah, it was a bit confusing for me too...

https://en.wikipedia.org/wiki/Doors_Open_Days
- "Doors Open Days"
- "Open House" (seems to be for the US)
- "Open Days"
- but also "Open Doors events in Wales"

I have a british coworker.. Maybe I can ask him.

Reply:
I was not aware that Open Doors Day is also possible. Maybe let's resolve it for now. It is a very minor issue. We can always rename it later if necessary.

Reply:
Ok. Can you finish the review, so we can merge?